### PR TITLE
Add german layout for redox keyboard

### PR DESCRIPTION
**Pull request by frederik-h**
A german layout for the redox keyboard which tries
to stay close to the default english redox layout
while acommodating the umlauts at their usual position.

### CODE REVIEW

File: keyboards/redox/keymaps/german/rules.mk [hunk 1]
```diff
@@ -0,0 +1,5 @@
+RGBLIGHT_ENABLE = yes
+
+ifndef QUANTUM_DIR
```
[the remark this answers is not in the document]
Comment (drashna) [reply keyboards/redox/keymaps/german/rules.mk]: No, that's fine.  The `ifndef QUANTUM_DIR` stuff. That line and the two after it, as well.

**Comment by frederik-h** [comment]
@drashna I have added the requested changes.

File: keyboards/redox/keymaps/german/config.h [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "../../config.h"
```
Comment (drashna) [inline keyboards/redox/keymaps/german/config.h]: This line isn't needed.  All relevant config.h files are included by default.

File: keyboards/redox/keymaps/german/keymap.c [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    XXXXXXX, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,                              KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  XXXXXXX,
+    XXXXXXX, RESET  , RGB_M_P, RGB_TOG, RGB_MOD, RGB_HUD, RGB_HUI,                            RGB_SAD, RGB_SAI, RGB_VAD, RGB_VAI, XXXXXXX, KC_DEL,  XXXXXXX,
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, _______,                            _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                            XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                            XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
+  )
+
+};
+
+#ifdef AUDIO_ENABLE
```
Comment (drashna) [inline keyboards/redox/keymaps/german/keymap.c]: It looks like this line and everything below it isn't needed, actually.

**Comment by frederik-h** [comment]
@drashna I have made the requested changes

**Comment by drashna** [comment]
Awesome, looks good!

**Comment by frederik-h** [comment]
Thanks for the review!

**Pull request merged**